auth/HashingWrappers.m

```diff
@@ -20,5 +20,15 @@
             output = typecast(m_a_c.doFinal(input), 'uint8');
         end
 
+        function key_chain = truncate_sha256(input, length)
+            input = HashingWrappers.sha_256(input);
```

Comment:
change function name to have underscore ahs_256...make consistent with other functions names